indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -127,6 +127,48 @@ function EditableListDisplay({
     [editors]
   );
 
+  const filterOptions = useMemo(
+    () => [
+      {
+        key: 'state',
+        text: Translate.string('Status'),
+        options: [
+          {value: 'accepted', text: Translate.string('Accepted')},
+          {value: 'ready_for_review', text: Translate.string('Ready for review')},
+          {value: 'not_submitted', text: Translate.string('Not submitted')},
+        ],
+      },
+      {
+        key: 'editor',
+        text: Translate.string('Editor'),
+        options: _.uniqBy(
+          contribsWithEditables.map(c => c.editable.editor).filter(Boolean),
```

Comment:
I think `x => x` is clearer than `Boolean`, but since you're refactoring things anyway I'll merge this one as it is.

Reply:
Thanks - fixed it in the new PR